graphql_api/tests/test_repository.py

```diff
@@ -116,16 +111,22 @@ def test_when_repository_has_no_coverage(self):
             repository_id=repo.repoid, token_type="profiling"
         ).key
         graphToken = repo.image_token
-        assert self.fetch_repository(repo.name) == {
+        assert self.fetch_repository(
+            repo.name,
+            default_fields
+            + "coverageAnalytics { percentCovered commitSha hits misses lines },",
```

Comment:
Is it possible to add the update the default_fields object with this new object instead of removing it and supplementing the query here?

Or would that have meant we needed to update more tests or something

Reply:
I thought these fields aren't necessarily "default" as we extend the repository object with more stuff and maybe more product lines into the future. I'd guess net new tests would more likely want to test repository without coverageAnalytics related fields (& those could get tested within the test_coverage_analytics.py area)